src/pages/gen1/[platform]/build-a-backend/storage/import/index.mdx

```diff
@@ -67,8 +67,21 @@ By default, Amplify Libraries assumes that S3 buckets are configured with the fo
 - `protected/{user_identity_id}/` - Readable by all users, but writable only by the creating user
 - `private/{user_identity_id}/` - Only accessible for the individual user
 
+<InlineFilter filters={[
+    'android',
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+]}>
+
 You can either configure your IAM role to use the Amplify-recommended policies or in your Amplify libraries configuration [overwrite the default storage path behavior](/gen1/[platform]/build-a-backend/storage/configure-access/#customize-object-key-path).
```

Comment:
why is flutter omitted here?

Reply:
It's removed from flutter because it links to the File Access Level page that no longer exists in v2.